explorer/src/lib/containers/statistics-panel/StatisticsPanel.svelte

```diff
@@ -26,61 +32,83 @@
     return value >= 1e6 ? millionFormatter(value) : valueFormatter(value);
   };
 
-  const statistics = [
+  const dataStore = createDataStore(duskAPI.getNodeLocations);
+  const pollingMarketDataStore = createPollingDataStore(
```

Comment:
Does this need to be polled? And if yes: with the same frequency of the main polling about blocks / transactions?